static/styles/drafts.css

```diff
@@ -69,6 +69,15 @@
             color: hsl(0, 0%, 67%);
             pointer-events: none;
         }
+
+        /* Hide default scrollbar for IE, Edge and Firefox */
+        -ms-overflow-style: none;
+        scrollbar-width: none;
+    }
+
+    /* Hide default scrollbar for Chrome, Safari and Opera */
+    .drafts-list::-webkit-scrollbar {
+        display: none;
```

Comment:
Why are these needed?  I'd expect simplebar to take car of the scrollbar situation entirely.